chapter_appendix-mathematics-for-deep-learning/naive-bayes.md

```diff
@@ -168,11 +168,29 @@ By itself, this expression does not get us any further. We still must estimate r
 
 $$ \hat{y} = \mathrm{argmax}_y \> \prod_{i=1}^d p(x_i  \mid  y) p(y).$$
 
-If we can estimate $\prod_i p(x_i=1  \mid  y)$ for every $i$ and $y$, and save its value in $P_{xy}[i, y]$, here $P_{xy}$ is a $d\times n$ matrix with $n$ being the number of classes and $y\in\{1, \ldots, n\}$. In addition, we estimate $p(y)$ for every $y$ and save it in $P_y[y]$, with $P_y$ a $n$-length vector. Then for any new example $\mathbf x$, we could compute
-
-$$ \hat{y} = \mathrm{argmax}_y \> \prod_{i=1}^d P_{xy}[x_i, y]P_y[y],$$
+If we can estimate $\prod_i p(x_i=1  \mid  y)$ for every $i$ and $y$, and save its value in $P_{xy}[i, y]$, here $P_{xy}$ is a $d\times n$ matrix with $n$ being the number of classes and $y\in\{1, \ldots, n\}$, i.e.,
+
+
+$$ 
+P_{xy}[i, y] = 
+\begin{cases}
+    P_{xy}[i, y] & \text{for } t_i=1 ;\\
+    1 - P_{xy}[i, y] & \text{for } t_i = 0 .
+\end{cases}
+$$
+
+In addition, we estimate $p(y)$ for every $y$ and save it in $P_y[y]$, with $P_y$ a $n$-length vector. Then for any new example $\mathbf x$, we could compute
+
+$$ 
+\begin{aligned}
+\hat{y} 
+&= \mathrm{argmax}_y \ \prod_{i=1}^d P_y[y] \times P_{xy}[i, y]\\
+&= \mathrm{argmax}_y \ \prod_{i=1}^d P_y[y]\ P_{xy}[i, y]^{t_i}\, \left(1 - P_{xy}[i, y]\right)^{1-t_i},
+\end{aligned}
+$$
```

Comment:
Hey @particle1331 , apology for a bit delay. Here is the reason why this "eqlabel" is not render well: 

"$$\begin{aligned} ... \end{aligned}$$" needs to be on one single line. 

Please check (search `\begin{aligned}`) in this example: https://github.com/d2l-ai/d2l-en/blob/master/chapter_linear-networks/linear-regression.md. 

Let me know if you are still having trouble solving it.

Reply:
Hi! Sorry for the delay as well. Thank you for pointing me towards this solution. 
It worked. :=) 

http://preview.d2l.ai/d2l-en/PR-1656/chapter_appendix-mathematics-for-deep-learning/naive-bayes.html